### ISSUE TITLE
Disabling ad-blocking from home screen removes all content

### ISSUE THREAD

**Issue by balboah** [blokadaorg/vpn-beta-tracker#25]
When connected to real VPN, I home Home screen and "Turn off" on "X domains blocked" popup.

All content gets removed until app is restarted

**Comment by kar** [comment]
can you rephrase

**Comment by balboah** [comment]
- Open home screen
- Expand "123 domains blocked"
- Click Turn off

All content is now removed, empty background and side navigation still works (but no content)
